packages/frontend/components/shared/solution/Solution.tsx

```diff
@@ -85,14 +90,27 @@ const SolutionDetails: React.FC<SolutionProps> = ({
         </div>
       </section>
```

Comment:
uporządkuj proszę przy okazji tę całą sekcję - wydziel z tego trochę tych komponentów UI, bo to wygląda okrponie :)